test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -36,17 +142,14 @@ public void Shutdown()
     }
 
     [TestFixture]
-    public abstract class RevitSystemTestBase : SystemTestBase
+    public class RevitSystemTestBase : SystemTestBase
     {
-        #region public methods
+        private string samplesPath;
+        private string defsPath;
+        protected string emptyModelPath1;
+        protected string emptyModelPath;
 
-        [SetUp]
-        public override void Setup()
-        {
-            base.Setup();
-
-            DocumentManager.Instance.CurrentUIApplication.ViewActivating += CurrentUIApplication_ViewActivating;
```

Comment:
@ikeough 
Is this required to fix something? I have an impression ViewActivating event is used for multiple test cases from a test source.